docs/extras/modules/memory/agent_with_memory.ipynb

```diff
@@ -86,7 +86,7 @@
    "id": "0021675b",
    "metadata": {},
    "source": [
-    "We can now construct the LLMChain, with the Memory object, and then create the agent."
+    "We can now construct the `LLMChain`, with the memory object, and then create the agent."
```

Comment:
The same. I stop here to make the same comments. Many changes below about changing proper names of the classes to the generic words. Please, rollback them.